frontend/src/pages/DashBoard/DashBoard.tsx

```diff
@@ -0,0 +1,50 @@
+import { ChangeEvent, useEffect, useState } from 'react';
+import UsersList from 'components/molecules/UsersList';
+import { UserRequests } from 'Services';
+import { useHistory } from 'react-router-dom';
+import {
+  StyledBottomWrapper,
+  StyledButton,
+  StyledContainer,
+  StyledInput,
+  StyledRightWrapper,
+  StyledTitle,
+  StyledTopWrapper,
+} from './DashBoard.styled';
+
+const DashBoard = (): JSX.Element => {
+  const history = useHistory();
+  const [users, setUsers] = useState([]);
```

Comment:
You are not doing anything with these users inside this component. Could we just move them into UsersList component and drop one props?